packages/preset-ant-design/package.json

```diff
@@ -20,13 +20,13 @@
   },
   "devDependencies": {
     "@babel/core": "^7.8.7",
-    "@storybook/react": "^5.3.17",
-    "@types/node": "^13.9.1",
+    "@storybook/react": "^6.1.17",
+    "@types/node": "^14.14.25",
     "@types/react": "^16.9.23",
-    "@types/storybook__react": "^3.0.9",
+    "@types/storybook__react": "^5.2.1",
```

Comment:
`@types/storybook__react` can be removed as `@storybook/react` is shipping its own type now

Reply:
Well the removal caused other subpackages to break as they still use @storybook/react 5.3. Should I bump those as well?